pkg/syndesis/operation/attach.go

```diff
@@ -0,0 +1,96 @@
+package operation
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	"github.com/syndesisio/syndesis-operator/pkg/apis/syndesis/v1alpha1"
+	"github.com/syndesisio/syndesis-operator/pkg/syndesis/template"
+	"github.com/syndesisio/syndesis-operator/pkg/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func AttachSyndesisToResource(syndesis *v1alpha1.Syndesis) error {
+
+	resTypes, err := getAllManagedResourceTypes()
+	if err != nil {
+		return err
+	}
+	selectors := getAllManagerSelectors()
+
+	for _, selector := range selectors {
+		for _, metaType := range resTypes {
+
+			options := sdk.WithListOptions(&selector)
+			list := metav1.List{
+				TypeMeta: metaType,
+			}
+			if err := sdk.List(syndesis.Namespace, &list, options); err != nil {
+				return err
+			}
+
+			for _, obj := range list.Items {
+				res, err := util.LoadKubernetesResource(obj.Raw)
+				if err != nil {
+					return err
+				}
+				SetNamespaceAndOwnerReference(res, syndesis)
+				if err := sdk.Update(res); err != nil {
+					return err
+				}
+			}
+
+		}
+	}
+
+	return nil
+}
+
+func getAllManagerSelectors() []metav1.ListOptions {
+	return []metav1.ListOptions {
+		{
+			LabelSelector: "syndesis.io/app=syndesis,syndesis.io/type=infrastructure",
+		},
+		{
+			LabelSelector: "app=syndesis,syndesis.io/app=todo",
+		},
+		{
+			LabelSelector: "syndesis.io/app=syndesis,syndesis.io/component=syndesis-db",
```

Comment:
going to fix this in syndesis, too (and the template), so maybe add a comment that this can be removed soon ?